pkg/controllers/pod/pod_controller.go

```diff
@@ -148,6 +150,67 @@ func (r *PodReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 	return common.ResultNormal, nil
 }
 
+// Set PodNetworkReady condition to false in Pod to inform spherelet the SubnetPort is restored with another vif ID
+func setReconfigureNicStatus(client client.Client, ctx context.Context, obj client.Object) error {
+	pod := obj.(*v1.Pod)
+	newConditions := []v1.PodCondition{
+		{
+			Type:               v1.PodConditionType("PodNetworkReady"),
```

Comment:
This condition is only used for podRestore?  we may change  PodNetworkReady to PodNetworkRestore for more specific. 
Who is responsible to set this condition to true? We only see setting is as False. 

For this change, please add Spherelet team to have a review.